ui/components/Navbar/Navbar.js

```diff
@@ -238,6 +244,50 @@ const Navbar = ({
     });
   };
 
+  const version = (

```

Comment:
Declaring this component as a const does not make sense here. If there is no specific reason for doing this, will you please remove it and just render the function inline ?

Reply:
Inside the main return, right??  @sudo-NithishKarthik

Reply:
Yep 👍